person.rb

```diff
@@ -1,25 +1,33 @@
+require_relative './nameable'
+
 class Person
-  att_accessor :id, :name, :age
-  def initialize(name='Unknown', age, parent_permission=false)
+  def initialize(age, name = 'Unknown', parent_permission: true)
     self.age = age
     self.name = name
     @parent_permission = parent_permission
     @id = Random.rand(1...100)
   end
 
-  private_class_method :is_of_age
-    if (@age >= 18)
-      return true
-    else
-      return false
-    end
+  attr_accessor :name, :age
```

Comment:
- [ ] kindly initialize the @rentals array in the constructor like the previous reviewer noted, this will be needed for the many-to-many relationship
- [ ] also, please like the previous reviewer also mentioned, create a method for adding rentals. The method can take 2 parameters: book and date. Afterwards, you instantiate a new Rental and pass in the date, book, and person as self to its constructor.